src/main/java/org/broadinstitute/barclay/argparser/CommandLineArgumentParser.java

```diff
@@ -69,6 +69,9 @@
     public static final String COMMENT = "#";
     public static final String POSITIONAL_ARGUMENTS_NAME = "Positional Argument";
 
+    // Extension for collection argument list files
+    private static final String COLLECTION_LIST_FILE_EXTENSION = ".list";
```

Comment:
I agree that this functionality needs to be discoverable somehow... Adding it to every Collection usage seems redundant, but maybe maybe not?  The help output definitely has to include is as an option somewhere, either in the preamble or on every collection argument.